src/Functions/largestConnCompSliceWise.m

```diff
@@ -1,6 +1,21 @@
-% Andreas Husch
-% 2018
 function binaryMask = largestConnCompSliceWise(img3d)
+%
```

Comment:
There is a description missing here.

Reply:
Right! There was no description provided in the original file. Could you give us a description of what this function does, @adhusch ?

Reply:
Please see my comment from 2 days ago

Reply:
I will include this in the description, then, thank you.